derrida/settings.py

```diff
@@ -143,8 +143,13 @@
 ]
 
 # django-compressor settings
+# COMPRESS_PRECOMPILERS = (
+#     ('text/x-scss', 'sass --scss --quiet {infile} {outfile}'),
+# )
+
+# Use local node-sass installed via npm
 COMPRESS_PRECOMPILERS = (
-    ('text/x-scss', 'sass --scss --quiet {infile} {outfile}'),
+    ('text/x-scss', 'node_modules/node-sass/bin/node-sass {infile} {outfile}'),
```

Comment:
I'm concerned about making this change in production because I don't want to have to deal with changing dependencies / setup / installation.

I'd like to move this to dev / ci settings (since it seems like that's where we need it), and make the node-sass npm package a dev dependency so that we have the fewest changes possible in our production environment.

Reply:
I will revert this since travis isn't operational anymore. I will shift this to `local_settings.py`